activities/tool-management/index.md

```diff
@@ -19,11 +19,17 @@ This guidance covers:
 
 Our current tools are:
 
+* [1password](1password.md) (password management)
 * [Docusign](docusign.md) (contract management)
+* [GitHub](github.md) (version control and codebase management)
 * [Google Workspace](google-workspace.md) (email, calendaring and internal document management)
 * [Jitsi Meet](jitsi-meet.md) (videoconferencing)
+* [Mailman](mailing-lists.md) (mailing lists with Hyperkitty archiver)
 * [Mattermost](mattermost.md) (chat)
+* [Miro](miro.md) (online collaborative whiteboard)
+ [Nextcloud](nextcloud.md) (file hosting services)
 * [Odoo Community](odoo.md) (CRM and project management)
```

Comment:
I suggest spelling out what the software does in stead of using CRM as 'contact relationship management', since both other _C_'s commonly used don't fit (consumer or constituent).

```suggestion
* [Odoo Community](odoo.md) (contact relationship management and project management)
```